markdown/events/2022/bytesize-inkscape.md

```diff
@@ -25,3 +25,93 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[20:05](https://www.youtube.com/watch?v=0vKhfedYKGo&t=1205)
+You get the point. An important thing to emphasize, as you may have already noticed, I'm just copying pasting. Like I said, 80 % of my Inkscape work is copying and pasting components I've already made. That is why we made this cheat sheet. I highly recommend you be using this as well, but of course you can modify them in whatever way you want. To get a basic outline I'd recommend trying something like this and then to modify. If I don't like this green color I need to ungroup this. I want a different color I can change this to a red. I'm gonna appear I would like this to have a purple on this one to be blue. Just keep working on this and construct things in this very lego-like fashion, that is my main recommendation. In this case I can then also add a station name. I'll call it input, put this up here, put this down here. Say - Step one. You want to change the fonts of your text. For example if you want to use the nf-core font. We have the font called `maven pro`. You can set this here as well.
+
+[21:22](https://www.youtube.com/watch?v=0vKhfedYKGo&t=1282)
+The final thing is. Maybe once you've completed your image, you want to not have this weird A4 sized page. You can also go to `File`, `Document Properties` and modify the layout in this way. For example `Setting` is landscape and also you `resize the page to content` if I press this button here. You can see that the page has been resized to cover all the different drawings that you may have in the particular document.
+
+[21:55](https://www.youtube.com/watch?v=0vKhfedYKGo&t=1315)
+This is pretty much it! The one other thing I would recommend doing before actually starting such a diagram is already have a working doodle or diagram of your pipeline. Sometimes this can get a bit fiddly when you're trying to work out exactly how everything should be spaced. For example for nf-core taxprofiler, we used google drawings and sketched out already all of the different parts of the pipeline. It's a bit easier to move things around here by snapping them together. Then you can use this as a reference to how to make your final work for documentation. You could take the example at the top. I said is the input. I have the two splits here then I'd set up like this and follow along the guidelines. I do that here.
```

Comment:
```suggestion
This is pretty much it! The one other thing I would recommend doing before actually starting such a diagram is already having a working doodle or diagram of your pipeline. Sometimes this can get a bit fiddly when you're trying to work out exactly how everything should be spaced. For example for nf-core/taxprofiler, we used google drawings and sketched out already all of the different parts of the pipeline. It's a bit easier to move things around here by snapping them together. Then you can use this as a reference to how to make your final work for documentation. You could take the example at the top. I said is the input. I have the two splits here then I'd set them up like this and follow along the guidelines. I do that here.
```